lib/c/criu.c

```diff
@@ -541,6 +541,17 @@ void criu_set_shell_job(bool shell_job)
 	criu_local_set_shell_job(global_opts, shell_job);
 }
 
+void criu_local_set_orphan_pts_master(criu_opts *opts, bool orphan_pts_master)
+{
+	opts->rpc->has_orphan_pts_master	= true;
+	opts->rpc->orphan_pts_master		= orphan_pts_master;
```

Comment:
This looks like a tab in front of the '='. That is unusual. Normally there are spaces.